app/src/main/res/values/styles.xml

```diff
@@ -178,4 +178,104 @@
     <item name="android:windowIsFloating">false</item>
     <item name="android:backgroundDimEnabled">true</item>
   </style>
+
+  <style name="H1Header" parent="Widget.AppCompat.TextView">
```

Comment:
Also, I'm not sure about this name convention seems a bit odd to me and would be hard to recall it (could be a minor inconvenience in some cases), we could simply heading1, heading2 and so on

Reply:
I have updated the code to make use of common style. 
Regarding naming: These names are suggested by Chantel and I think in either case its just about habit, once you get used to it both becomes easy else in both case the contributor will need to check attributes to understand things.

Reply:
I know that is just a habit nothing else but it still looks a bit odd to me maybe ben could suggest some better nomenclature for this otherwise LGTM.

Reply:
I have updated names to heading as both seems same to me for now.